sysutils/freeradius/Portfile

```diff
@@ -27,20 +30,20 @@ long_description        The FreeRADIUS Server Project is an attempt to create \
                         RADIUS server, but they don't share a lot in common \
                         anymore.
 
-master_sites            ftp://ftp.freeradius.org/pub/freeradius/ \
-                        ftp://ftp.freeradius.org/pub/freeradius/old
+openssl.branch          1.1
 
-depends_lib             port:libtool \
+depends_lib-append      port:libtool \
                         port:libpcap \
                         path:lib/libldap.dylib:openldap \
-                        path:lib/libssl.dylib:openssl \
                         port:talloc \
-                        port:gdbm
+                        port:gdbm   \
+                        port:pcre   \
+                        port:perl5
```

Comment:
Thank you for looking into this, but I’m not sure whether depending on the `perl5` works for ports which link to Perl rather than only running it as a command. I think the problem may still exist if I were to have a different variant of `perl5` than the one this port was built with, and do not have the perl5.xx port it linked to.

Reply:
Sure.  Any recommendations come to mind?

Reply:
These changes appear to work: https://github.com/chrstphrchvz/macports-ports/commit/0d4ded03ea0be5c9be09b3f0a418b7bf39b8f910

Reply:
Latest Perl 5 is 5.34.0, should we be using that version?

Reply:
We should be using whatever the default version of perl is in the perl5 portgroup. Today, that's 5.28.